synapse/models/risk.py

```diff
@@ -144,6 +147,25 @@ def getModelDefs(self):
                     }),
                 )),
 
+                ('risk:alert', {}, (
```

Comment:
Do we want to track what generated the alert? For example this software/appliance/host sent this alert?

Reply:
For flexibility, i'd prefer to use light edges for this for a while as well until we know what fits best :)